logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -213,6 +213,7 @@
                 <field name="origin"/>
                 <field name="destination_partner_id"/>                
                 <field name="picking_id"/>
+                <field name="stock_move_id" string="Ticket/Mov."/>
```

Comment:
Even if new legend translated to `es` language will be the same, we should update translation files, or this string will appear in a later commit.